src/components/SectionHeader/_SectionHeader.scss

```diff
@@ -1,5 +1,4 @@
-.coa-SectionHeader {
-  text-transform: uppercase;
+.coa-SectionHeader h2 {
```

Comment:
more specificity here that using BEM is supposed to help you avoid. Not sure why we need to add the `h2` but my personally preference would be to change the classname or add a modifier to this one like `coa-SectionHeader--large-text` 🤷‍♂️

Reply:
Not sure I understand. h2 is a child of .coa-SectionHeader. I can add an additional class to the h2, if we want, and reference that, I just didn't see the need, since we'll always require h2s as a child of section headers. Maybe you thought this was h2.coa-SectionHeader?